components/shared/AuthorFacePile.tsx

```diff
@@ -62,6 +62,9 @@ export default function AuthorFacePile({
         Boolean(horizontal) && styles.horizontal,
         overrideStyle
       )}
+      // When negative margins are used, the width of this container will be smaller than actually is and result in unintended spacing of adjacent elements.
```

Comment:
I've seen us have these problems with a lot of other components. Feels like we need to clean up some css in some spots so these weird spacings don't happen